subsys/bluetooth/host/gatt.c

```diff
@@ -3408,9 +3428,9 @@ static int read_included_uuid(struct bt_conn *conn,
 {
 	BT_DBG("handle 0x%04x", params->_included.start_handle);
 
-	return gatt_req_send(conn, read_included_uuid_cb, params,
-			     read_included_uuid_encode, BT_ATT_OP_READ_REQ,
-			     sizeof(struct bt_att_read_req));
+	return gatt_req_send(conn, read_included_uuid_cb, params, read_included_uuid_encode,
```

Comment:
Yes, autoformatting. The `ColumnLimit` is 100 in `.clang-format`, so that is automatically enforced. I don't think clang-format understands "80 columns is recommended".

Reply:
In any case, it is fine that they exceed 80, but it shouldn't automatically move code to always attempt to use the full 100, as that is not only disruptive (modifies lines of code that the PR otherwise doesn't change), which makes it harder to do `git blame`. 

I've seen a lot of issues with people using auto-clang-format, so this isn't the first time I've seen it doing stuff it shouldn't :D The Zephyr (and Linux) code style guide is not properly support for clang format unfortuntately.

Reply:
Ok, will change this to make the diff minimal.